### PR TITLE
diagnostic-connection-for-database-administrators with SSMS

### PR DESCRIPTION
**Pull request by ChadSimmons**
Add steps for creating a DAC from SQL Server Management Studio to address the error "Dedicated administrator connections are not supported via SSMS as it establishes multiple connections by design. (Microsoft.SqlServer.Management.SqlStudio.Explorer)"

### CODE REVIEW

**Comment by PRMerger17** [comment]
@ChadSimmons : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by MikeRayMSFT** [comment]
Thank you @ChadSimmons - reviewing

File: docs/database-engine/configure-windows/diagnostic-connection-for-database-administrators.md [hunk 1]
```diff
@@ -36,7 +36,13 @@ ms.author: mikeray
...
+ The DAC is available and supported through the **sqlcmd** command-prompt utility using a special administrator switch (**-A**). For more information about using **sqlcmd**, see [Use sqlcmd with Scripting Variables](../../relational-databases/scripting/sqlcmd-use-with-scripting-variables.md). You can also connect prefixing **admin:** to the instance name in the format **sqlcmd -S admin:<*instance_name*>**. You can also initiate a DAC from a [!INCLUDE[ssManStudioFull](../../includes/ssmanstudiofull-md.md)] Query Editor by connecting to **admin:\<*instance_name*>**.
+ 
+> [!Note]  
+> To establish a DAC from SQL Server Management Studio
+> - Disconnect all connections to the related database
```
Comment (bluefooted) [inline docs/database-engine/configure-windows/diagnostic-connection-for-database-administrators.md:43]: > - Disconnect all connections to the related SQL Server instance, including the Object Explorer and all open query windows.

File: docs/database-engine/configure-windows/diagnostic-connection-for-database-administrators.md [hunk 2]
```diff
@@ -36,7 +36,13 @@ ms.author: mikeray
...
+ The DAC is available and supported through the **sqlcmd** command-prompt utility using a special administrator switch (**-A**). For more information about using **sqlcmd**, see [Use sqlcmd with Scripting Variables](../../relational-databases/scripting/sqlcmd-use-with-scripting-variables.md). You can also connect prefixing **admin:** to the instance name in the format **sqlcmd -S admin:<*instance_name*>**. You can also initiate a DAC from a [!INCLUDE[ssManStudioFull](../../includes/ssmanstudiofull-md.md)] Query Editor by connecting to **admin:\<*instance_name*>**.
+ 
+> [!Note]  
+> To establish a DAC from SQL Server Management Studio
+> - Disconnect all connections to the related database
+> - From the menu select **File** > **New** > **Database Engine Query**
+> - From the connection dialog box in the Server Name field, enter **admin:** if using the default instance or **admin:\<*instance_name*\>** if using a named instance
```
Comment (bluefooted) [inline docs/database-engine/configure-windows/diagnostic-connection-for-database-administrators.md:45]: **admin:<*server_name*>** if connecting to a default instance or **admin:<*server_name*>\<*instance_name*>** if connecting to a named instance.

**Comment by bluefooted** [comment]
I'm good with adding this, but I have requested a few changes to the wording, see my review above.

**Comment by MikeRayMSFT** [comment]
@ChadSimmons, feel free to incorporate the proposed changes, or let me know and I will. Thank you.

File: docs/database-engine/configure-windows/diagnostic-connection-for-database-administrators.md (hunk 1 above)
Comment (ChadSimmons) [reply docs/database-engine/configure-windows/diagnostic-connection-for-database-administrators.md:43]: Apologies for the delay.  GitHub notifications got marked as spam in an Outlook.com account (how is that possible!?).
Anyway, I would happily accept the change if I could figure out how.  I even phoned a friend and we're stumped.

File: docs/database-engine/configure-windows/diagnostic-connection-for-database-administrators.md (hunk 2 above)
Comment (ChadSimmons) [reply docs/database-engine/configure-windows/diagnostic-connection-for-database-administrators.md:45]: I would happily accept the change if I could figure out how.

File: docs/database-engine/configure-windows/diagnostic-connection-for-database-administrators.md (hunk 1 above)
Comment (bluefooted) [reply docs/database-engine/configure-windows/diagnostic-connection-for-database-administrators.md:43]: No worries! I think you need to rewrite your change to match my review, I don't think there's a way to automatically accept it.

Comment (MikeRayMSFT) [reply docs/database-engine/configure-windows/diagnostic-connection-for-database-administrators.md:43]: I can't figure it out. I'm going to manually take in the changes @bluefooted recommended in our private repository.

@ChadSimmons thanks for the submission.

**Comment by MikeRayMSFT** [comment]
I'm closing this pull request and publishing in https://github.com/MicrosoftDocs/sql-docs-pr/pull/14735

We couldn't submit the update because the clone that was used for the original pull request is not available. Don't know why. I just manually added it.

**Pull request closed without merging**